apps/web/src/pages/Pools/PoolAdvertisementPage/components/DeadlineDialog.tsx

```diff
@@ -30,8 +30,7 @@ interface DeadlineDialogProps {
 const DeadlineDialog = ({ deadline }: DeadlineDialogProps) => {
   const intl = useIntl();
   const routes = useRoutes();
-  // https://stackoverflow.com/a/34602679
-  const localTimeZone = Intl.DateTimeFormat().resolvedOptions().timeZone;
+  const localTimeZone = getLocalTimezone();
```

Comment:
This isn't needed anymore, right?  Probably the other use of this helper function isn't either?

Reply:
Yeah, rolled back in [remove local timezone helper](https://github.com/GCTC-NTGC/gc-digital-talent/pull/12469/commits/2e43b02a4eb6e06d63ebff954da6011725fc6b7e)